Source/AwsCommonRuntimeKit/io/TlsContextOptions.swift

```diff
@@ -2,86 +2,72 @@
 //  SPDX-License-Identifier: Apache-2.0.
 
 import AwsCIo
-//swiftlint:disable trailing_whitespace
 public final class TlsContextOptions {
+    private let allocator: Allocator
     var rawValue: UnsafeMutablePointer<aws_tls_ctx_options>
-    
     public static func isAlpnSupported() -> Bool {
         return aws_tls_is_alpn_available()
     }
-    
```

Comment:
Even if I put a tab here to remove this change, git does not detect that change. I think the IDE just reverts it automatically.